packages/components/src/internal/util/helpLinks.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import { HelpLink, getHelpLink } from './helpLinks';
+
+const HELP_LINK_BASE_URL = 'https://www.labkey.org/Documentation/wiki-page.view?';
+
+describe('HelpLink', () => {
+    test('default props', () => {
+        const component = <HelpLink topic="TEST_TOPIC">default props text</HelpLink>;
+        const tree = renderer.create(component).toJSON();
```

Comment:
nit: No need to call `.toJSON()` in these cases.

Reply:
that's right. thanks for the reminder. done